api/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.6.6
 	github.com/hashicorp/go-rootcerts v1.0.2
 	github.com/hashicorp/hcl v1.0.0
-	github.com/hashicorp/vault/sdk v0.1.14-0.20200519221838-e0cfd64bc267
+	github.com/hashicorp/vault/sdk v1.5.0
```

Comment:
This can't be correct ... the current latest version of the sdk is v0.7.0. There is no v1.5.0.

Reply:
@maxb that's what I thought too, but also... I'm often wrong. That's what I get for making an assumption the other way! :-D